guides/common/modules/snip_supported-client-operating-systems.adoc

```diff
@@ -7,6 +7,11 @@ ifndef::orcharhino[]
 endif::[]
 ifdef::satellite[]
 * {EL} 7 and 6 with the https://www.redhat.com/en/resources/els-datasheet[ELS Add-On]
```

Comment:
Starting July RHEL 6 is also dead: https://access.redhat.com/articles/4665701

Reply:
There was some internal discussion and it may not be entirely dead for all customers.